ztp/extra-manifests-builder/05-kdump-config/kdump-remove-ice-module.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# This script removes the ice module from kdump to prevent kdump failures on certain servers.
+# This is a temporary workaround for RHELPLAN-138236 and can be removed when that issue is
+# fixed.
+
+set -x
+
+SED="/usr/bin/sed"
+GREP="/usr/bin/grep"
+
+# override for testing purposes
+KDUMP_CONF="${1:-/etc/sysconfig/kdump}"
+REMOVE_ICE_STR="module_blacklist=ice"
+
+# exit if file doesn't exist
+[ ! -f ${KDUMP_CONF} ] && exit 0
```

Comment:
Normally we want to `exit 1` on error? Not sure how important that is in this case, though?